plenum/cli/cli.py

```diff
@@ -134,12 +134,14 @@ def __init__(self, looper, basedirpath, nodeReg=None, cliNodeReg=None,
                 and len(cliNodeReg)):
             self.nodeRegLoadedFromFile = True
             dataDir = self.basedirpath
-            ledger = Ledger(CompactMerkleTree(hashStore=FileHashStore(
-                dataDir=dataDir)),
+            fileHashStore = FileHashStore(dataDir=dataDir)
+            ledger = Ledger(CompactMerkleTree(hashStore=fileHashStore),
                 dataDir=dataDir,
                 fileName=self.config.poolTransactionsFile)
             nodeReg, cliNodeReg, _ = TxnStackManager.parseLedgerForHaAndKeys(
                 ledger)
+            ledger.stop()
```

Comment:
As I can see, Ledger's __init__ method just gets an already created Merkle tree. Ledger itself does not create a FileHashStore for the Merkle tree, so closing the FileHashStore of the Merkle tree is not Ledger's responsibility as well.